src/Stripe.net/Entities/Plans/PlanTier.cs

```diff
@@ -4,6 +4,9 @@ namespace Stripe
 
     public class PlanTier : StripeEntity
     {
+        [JsonProperty("flat_amount")]
+        public long FlatAmount { get; set; }
```

Comment:
This should be nullable.

(`UnitAmount` should be nullable too btw, but I guess that'd be a breaking change, so let's fix in the next major version.)

Reply:
Well, a user just reported a breakage because `UnitAmount` is not nullable (#1375) so let's try and fix that ASAP.

Reply:
Well this is going in a breaking release already so fixing now.